charts/backstage/values.yaml

```diff
@@ -71,6 +111,15 @@ upstream:
   ingress:
     host: "{{ .Values.global.host }}"
 
+  extraDeploy:
+    - apiVersion: v1
+      data:
+        dynamic-plugins.yaml: >
+          {{- include "common.tplvalues.render" ( dict "value"
+          .Values.global.dynamicPlugins "context" $) | nindent 4 }}
+      kind: ConfigMap
+      metadata:
+        name: dynamic-plugins
```

Comment:
Can we move this into the `charts/backstage/templates/` folder as a standalone template?

Reply:
Done.